swarm/main.cpp

```diff
@@ -19,13 +19,12 @@
 #include <options/options.hpp>
 #include <raft/raft.hpp>
 #include <storage/storage.hpp>
-
 #include <boost/log/expressions.hpp>
 #include <boost/log/support/date_time.hpp>
 #include <boost/log/utility/setup/common_attributes.hpp>
 #include <boost/log/utility/setup/file.hpp>
 #include <boost/program_options.hpp>
-
+#include <boost/filesystem.hpp>
```

Comment:
We should be using std::filesystem throughout our code.

Reply:
I agree, but until Apple get's it's poop together, I think we'll need boost. I will write a story in Jira about upgrading as soon as we can.